apps/website/src/pages/animal-quest/[episodeName].tsx

```diff
@@ -502,6 +515,37 @@ const AnimalQuestEpisodePage: NextPage<AnimalQuestEpisodePageProps> = ({
           </Consent>
         </Section>
       </div>
+
+      <JsonLD<Episode>
```

Comment:
@pjeweb maybe you have an idea here -- this would not work for me inside `<Meta>`, React/Next just didn't render it, which felt very odd?

Reply:
Well that's even weirder

Reply:
yeah no clue. it just get nuked when the head gets rendered. and you cannot use the `<Script strategy="beforeInteractive" />` tag since that is only allowed in the layout/document (https://nextjs.org/docs/messages/no-before-interactive-script-outside-document)

Reply:
did you try to deploy the version with it in the header and run the tests?

Reply:
Negative, just building locally. It always seemed to disappear as soon as JS runs 🤷